### PR TITLE
Thunderbird: update to 78.8.1

### PR DESCRIPTION
**Pull request by cgrzemba**
This updates Thunderbird version 78.8.1 and introduce similar changes like in Firefox

### CODE REVIEW

**Comment by AndWac** [comment]
You also got 2 commits for firefox into this PR for thunderbird!

File: components/mail/thunderbird/manifests/sample-manifest.p5m [hunk 1]
```diff
@@ -22,9026 +22,121 @@ set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
...
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDeviceContextSpecProxy.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDeviceSensors.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDirPrefs.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDirection.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDirectoryService.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDirectoryServiceAtomList.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDirectoryServiceDefs.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDirectoryServiceUtils.h
-file path=usr/include/thunderbird-$(COMPONENT_VERSION)/nsDisplayItemTypes.h
-file path=usr/include/thunderbird-$(COMPONEN
```
Comment (AndWac) [inline components/mail/thunderbird/manifests/sample-manifest.p5m]: Is the sample-manifest freshly generated? In my version all usr/lib/thunderbird/... paths contain $(MACH64) making them usr/lib/$(MACH64)/thunderbird.
This is also what I expect with BUILD:BITS=64.

**Comment by cgrzemba** [comment]
I changed the lightning retiring style.

**Comment by AndWac** [comment]
You need to update the meta data in pkg5. After the last commit the thunderbird-lightning package has also been removed from this file.

**Comment by cgrzemba** [comment]
pkg5 updated

**Pull request merged**